### PR TITLE
google-java-format: 0.1-alpha

### PR DESCRIPTION
**Pull request by scottbessler**
add formula for google-java-format tool

### CODE REVIEW

File: Library/Formula/google-java-format.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class GoogleJavaFormat < Formula
+  desc "Tool to format java code with the google java format"
+  homepage "https://github.com/google/google-java-format"
+  url "https://github.com/google/google-java-format/releases/download/google-java-format-0.1-alpha/google-java-format-0.1-alpha.jar"
+  version "0.1-alpha"
```
Comment (mikemcquaid) [inline Library/Formula/google-java-format.rb]: This needs to be a stable release (e.g. non-alpha/non-beta) to be included in Homebrew.

File: Library/Formula/google-java-format.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  bottle :unneeded
+
+  def install
+    libexec.install "google-java-format-#{version}.jar"
+    bin.write_jar_script libexec/"google-java-format-#{version}.jar", "google-java-format"
+  end
+
+  test do
+    system "#{bin}/google-java-format", "-v"
```
Comment (mikemcquaid) [inline Library/Formula/google-java-format.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/google-java-format.rb (hunk 1 above)
Comment (scottbessler) [inline Library/Formula/google-java-format.rb]: gotcha, so its better if i make a separate tap for this kind of thing?

Comment (mikemcquaid) [inline Library/Formula/google-java-format.rb]: Think that would be best for now, thanks!

Comment (DomT4) [inline Library/Formula/google-java-format.rb]: Can always submit to https://github.com/Homebrew/homebrew-devel-only if desired.

**Comment by DomT4** [comment]
Closing per:
> Can always submit to https://github.com/Homebrew/homebrew-devel-only if desired.

Thanks for the submission to Homebrew! :heart:

**Pull request closed without merging**